File: DSA/YashKrishanGupta_590014798/Day_33_Question_1.java
import java.util.Scanner;
public class Day_33_Question_1 {

    // Solution 1
    public static boolean oppositeSignCheck(int a ,int b) {
        return (a ^ b) < 0;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter two integers : ");
        int a = sc.nextInt();
        int b = sc.nextInt();
        System.out.println(oppositeSignCheck(a,b));

    }

File: DSA/PriyankaSingh_590016827/day33question1.c

#include <stdio.h>

int main() {
    int a, b;
    printf("Enter two integers: ");
    scanf("%d %d", &a, &b);

    if ((a ^ b) < 0)
        printf("True\n");
    else
        printf("False\n");

    return 0;
}
